tools/pd-ctl/tests/resourcemanager/resource_manager_command_test.go

```diff
@@ -68,7 +68,7 @@ func (s *testResourceManagerSuite) TestConfigController() {
 
 		actualCfg := server.ControllerConfig{}
 		err = json.Unmarshal(output, &actualCfg)
-		re.NoError(err)
+		re.NoError(err, string(output))
```

Comment:
why change this line?

Reply:
There is another flaky test: https://github.com/tikv/pd/issues/8035, I cannot reproduce it in my local environment.